index.js

```diff
@@ -18,34 +16,25 @@ var typedArrays = {
 };
 
 var slice = String.prototype.slice;
-var toStrTags = {};
 var gOPD = Object.getOwnPropertyDescriptor;
-if (hasToStringTag && gOPD && Object.getPrototypeOf) {
-	forEach(typedArrays, function (_, typedArray) {
-		var arr = new global[typedArray]();
-		if (!(Symbol.toStringTag in arr)) {
-			throw new EvalError('this engine has support for Symbol.toStringTag, but ' + typedArray + ' does not have the property! Please report this.');
```

Comment:
I'd still like to keep this error, so I can be alerted if an engine adds the Symbol without adding it to all the right places.